### PR TITLE
Feature/add css to teacher views

### PR DESCRIPTION
**Pull request by bryanmg**
## Description
- Add css for all views for teacher user type.

## Screenshots
image
image
image

### CODE REVIEW

File: app/views/students/attendances/_attendance_records_form.html.erb [hunk 1]
```diff
@@ -0,0 +1,39 @@
+<%= form_with(method: :get, url: url) do |form| %>
```
Comment (itsmeurbi) [inline app/views/students/attendances/_attendance_records_form.html.erb:1]: I'd say this and [_attendance_stats.html.erb](https://github.com/bryanmg/ammdk_colima/pull/43/files#diff-1589a313ff1c39edd9df4e6bc936335a1ecbeb8ac1c4ca79be2dbc45df99da26) changes had to belong to another PR 👀

File: app/controllers/students_learning_resources_controller.rb [hunk 2]
```diff
@@ -25,4 +25,8 @@ def set_user
   def assignee_params
     params.permit(:user_id, :learning_resource_id)
   end
+
+  def my_students_ids
+    @user.group_members.map(&:user_id)
```
Comment (itsmeurbi) [inline app/controllers/students_learning_resources_controller.rb:30]: `map` being an enum method will load and instantiate all `group_members` records. If you only care about `user_id`, then you might be using [pluck](https://apidock.com/rails/ActiveRecord/Calculations/pluck) instead

File: app/views/teachers/students/edit.html.erb [hunk 3]
```diff
@@ -1,8 +1,7 @@
 <div class="container">
-  <h1 class="subtitle" >Editing student</h1>
+  <br>
+  <h1 class="title" >Editing student</h1>
```
Comment (virgyjimenez) [inline app/views/teachers/students/edit.html.erb:3]:
```suggestion
  <h1 class="title" >Edit student</h1>
```

File: app/controllers/students_learning_resources_controller.rb (hunk 2 above)
Comment (itsmeurbi) [reply app/controllers/students_learning_resources_controller.rb:30]: Also I'm not sure if this is what you want, but looks like `@user` is a profesor and you want to get his/her students.
If that's the case, a nicer approach would be having a instance method responsible of getting that records. That way you can remove duplicity [here](https://github.com/bryanmg/ammdk_colima/pull/43/files#diff-24cbafec121d4311b5071b23b3cd02be297393a3a60dedb7c34173d435c4a627R55)
PLUS if `@user` actually represents a profesor, why don't we use the right var name?

e.g : `@profesor.students`

**Review by itsmeurbi — CHANGES_REQUESTED** [review]
You right, too many changes :)

File: app/views/teachers/learning_resources/edit.html.erb [hunk 4]
```diff
@@ -1,9 +1,5 @@
-<h1>Editing learning resource</h1>
+<div class="container pt-5">
+  <h1 class="title">Editing learning resource</h1>
```
Comment (virgyjimenez) [inline app/views/teachers/learning_resources/edit.html.erb:2]:
```suggestion
  <h1 class="title">Edit learning resource</h1>
```

**Review by virgyjimenez — CHANGES_REQUESTED** [review]

File: app/views/students/attendances/_attendance_stats.html.erb [hunk 5]
```diff
@@ -0,0 +1,32 @@
+<div id="attendance_stats">
+  <nav class="level">
+    <div class="level-item has-text-centered">
+      <div>
+        <p class="heading">Present</p>
+        <p class="title"><%= attendance_stats[true] ? attendance_stats[true] : 0 %></p>
```
[the remark this answers is not in the document]
Comment (bryanmg) [reply app/views/students/attendances/_attendance_stats.html.erb:6]: I'm no sure why is happen this, but fixed!

File: app/views/teachers/groups/_form.html.erb [hunk 6]
```diff
@@ -11,37 +11,93 @@
...
+  <div class="field is-horizontal">
+    <div class="field-label is-normal">
+      <%= form.label :days_of_the_week, class: "label" %>
+    </div>
+    <div class="field-body">
+      <div class="field pt-2">
+        <%= form.fields_for :days_of_the_week do |field| %>
+          <label class="checkbox mr-3">
+            Sunday
+            <%= field.check_box :sunday, { checked: group[:days_of_the_week][:sunday] == "1" ? true : false } %>
```
[the remark this answers is not in the document]
Comment (bryanmg) [reply app/views/teachers/groups/_form.html.erb:62]: Same here, I'm not sure why the code is overwritten but fixed, and applied advice

File: app/controllers/students_learning_resources_controller.rb (hunk 2 above)
Comment (bryanmg) [reply app/controllers/students_learning_resources_controller.rb]: I recently fixed the first comment, for the second one. I'll leave a TODO task.

File: app/views/students/attendances/_attendance_records_form.html.erb (hunk 1 above)
Comment (bryanmg) [reply app/views/students/attendances/_attendance_records_form.html.erb:1]: You're could be, but it is time to deploy :(

File: app/views/attendances/_attendance.html.erb [hunk 7]
```diff
@@ -1,11 +1,12 @@
...
+  <td>
     <%= attendance.user.name %>
-    <% if attendance.present %>
+  </td>
+  <td>
+    <% if attendance.present? %>
       <strong>present</strong>
     <% else %>
       <strong>absent</strong>
     <% end %>
```
Comment (luisAzcuaga) [inline app/views/attendances/_attendance.html.erb:10]: I think this would look cleaner :)
```erb
<strong>
<%= attendance.present? ? "present" : "absent" %>
</strong>
```

File: app/views/teachers/learning_resources/_learning_resource.html.erb [hunk 8]
```diff
@@ -1,22 +1,56 @@
...
+      <p>
+        <strong>Belt required:</strong>
+        <%= BELTS_TEXT[BELTS.key(learning_resource.belt)] %>
+      </p>
+
+      <div class="is-flex is-justify-content-flex-end mt-4 mr-3">
+        <%# <%= link_to(teacher_learning_resource_path(@user, learning_resource), class: "button is-primary is-outlined ml-5") do 
+          <i class="far fa-eye"></i>
+          <span class="ml-2">Show</span>
+        <% end %>
```
Comment (luisAzcuaga) [inline app/views/teachers/learning_resources/_learning_resource.html.erb:23]: Delete this if you won't use it anymore.

File: app/views/teachers/learning_resources/index.html.erb [hunk 9]
```diff
@@ -1,20 +1,27 @@
...
-  <% end %>
-</div>
+  <div>
+    <% @teacher_learning_resources.each do |learning_resource| %>
+      <%= render "teachers/learning_resources/learning_resource", learning_resource: learning_resource%>
+    <% end %>
+  </div>
 
-<%= will_paginate @teacher_learning_resources, renderer: RemoteLinkPagination::LinkRenderer, :class=>"" %>
+  <%= will_paginate @teacher_learning_resources, renderer: RemoteLinkPagination::LinkRenderer, :class=>"" %>
```
Comment (luisAzcuaga) [inline app/views/teachers/learning_resources/index.html.erb:10]:
```suggestion
  <%= will_paginate @teacher_learning_resources, renderer: RemoteLinkPagination::LinkRenderer %>
```

File: app/views/teachers/reviews/_form.html.erb [hunk 10]
```diff
@@ -12,23 +11,51 @@
...
   </div>
 
+  <div class="field is-horizontal">
+    <div class="field-label is-normal">
+      <%= form.label :learning_resource_id, class: "label" %>
+    </div>
+    <div class="field-body">
+      <div class="field">
+        <div class="select">
+          <%= form.select :learning_resource_id, @user.learning_resources&.pluck(:name, :id).prepend(["None", ""]), selected: review.learning_resource_id || ["None", ""] , class: "input"%>
```
Comment (luisAzcuaga) [inline app/views/teachers/reviews/_form.html.erb:48]: I'd change the `prepend` for the native [include_blank](https://api.rubyonrails.org/v5.1.7/classes/ActionView/Helpers/FormOptionsHelper.html)

File: app/views/teachers/students/index.html.erb [hunk 11]
```diff
@@ -1,17 +1,30 @@
...
+    <tbody>
+      <% @students.each do |student| %>
+        <tr>
+          <%= render "teachers/students/student", student: student, teacher: @user %>
+        </tr>
+      <% end %>
+    </tbody>
+  </table>
 
+  <%= will_paginate @students, renderer: RemoteLinkPagination::LinkRenderer, :class=>"is-flex is-justify-content-space-around" %>
```
Comment (luisAzcuaga) [inline app/views/teachers/students/index.html.erb:22]:
```suggestion
  <%= will_paginate @students, renderer: RemoteLinkPagination::LinkRenderer, class: "is-flex is-justify-content-space-around" %>
```

**Review by luisAzcuaga — CHANGES_REQUESTED** [review]

File: app/views/teachers/students/index.html.erb (hunk 11 above)
Comment (itsmeurbi) [reply app/views/teachers/students/index.html.erb:22]: @bryanmg why is this marked as resolved?

File: app/views/teachers/learning_resources/index.html.erb (hunk 9 above)
Comment (itsmeurbi) [reply app/views/teachers/learning_resources/index.html.erb:10]: Same here. Actually `class` attr is totally useless, isn't it?

**Comment by itsmeurbi** [comment]
@bryanmg why did you merged without any approval?
